src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,26 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersAllValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
+            limit: Joi.string(),
```

Comment:
Poderia ter tratado no próprio Joi, desse campo ser um número inteiro, e já com um valor padrão caso o usuário não informasse.
```suggestion
            limit: Joi
                .number()
                .integer()
                .positive()
                .max(50)
                .default(50),
```